packages/dotenv-webpack-plugin/src/index.js

```diff
@@ -0,0 +1,71 @@
+const fs = require("fs");
+const dotenv = require("dotenv");
+const dotenvExpand = require("dotenv-expand");
+const { DefinePlugin } = require("webpack");
+
+class DotenvWebpackPlugin {
+  constructor(config = {}) {
+    const currentDirectory = process.cwd();
+
+    this.config = Object.assign(
```

Comment:
Do we really `Object.assign`? If we will not modify `this.config` we don't need to do extra clone, it spends CPU and memory.

Also let's rename to `this.options`

Reply:
since this plugin needs to work standalone as well, we take input from user which we merge with inbuilt config using `Object.assign`
renamed variable to `options`